packages/browser-service/src/browser.service.ts

```diff
@@ -119,50 +147,67 @@ export class BrowserService {
     });
 
     await this.browserContextService.initBrowserContext();
+    await this.annotateExtensionWalletVersion(extensionService);
+    this.setWalletPage();
+    await this.getEOAWalletPage().setup(this.options.networkConfig.chainName);
```

Comment:
I think we can remove it and add     `await this.walletPage.options.extensionPage?.setup(
      this.options.networkConfig.chainName,
    );` with comment